sns/ut/parity_math_ut.c

```diff
@@ -1434,48 +1434,111 @@ static void ub_large_32768(int iter)
 	parity_math_tb();
 }
 
-enum { UB_ITER = 1 };
+static void ub_small_4_2_4K(int iter)
+{
+	UNIT_BUFF_SIZE = 4096;
+	duc = 4;
+	puc = 2;
+	fuc = duc+puc;
+	parity_math_tb();
+}
+
+static void ub_small_4_2_256K(int iter)
+{
+	UNIT_BUFF_SIZE = 262144;
+	duc = 4;
+	puc = 2;
+	fuc = duc+puc;
+	parity_math_tb();
+}
+
+static void ub_small_4_2_1M(int iter)
+{
+	UNIT_BUFF_SIZE = 1048576;
+	duc = 4;
+	puc = 2;
+	fuc = duc+puc;
+	parity_math_tb();
+}
+
+enum { UB_ITER = 100 };
 
 struct m0_ub_set m0_parity_math_ub = {
         .us_name = "parity-math-ub",
         .us_init = ub_init,
         .us_fini = NULL,
         .us_run  = {
-                { .ub_name  = "s 10/05/ 4K",
+                { .ub_name  = "s 10/05/4K",
                   .ub_iter  = UB_ITER,
-                  .ub_round = ub_small_4096 },
+                  .ub_round = ub_small_4096,
+                  .ub_block_size = 4096,
+				  .ub_blocks_per_op = 15 },
```

Comment:
Indentation.

And similar ones in the following.

Reply:
Done